backend/db/load.py

```diff
@@ -144,43 +144,47 @@ def indexes_and_derived_tables(
 
     # Read DDL
     print('INFO: Creating derived tables (e.g. `all_csets`) and indexes.')
+    # todo: Improve so that it iterates over modules & statements, rather than reaading in all modules,
+    #  and concatenating into a list of statements.
     statements = get_ddl_statements(schema=schema_name)
 
     # Determine which steps still needed
     if start_step:
         last_successful_step = start_step
     else:
-        with get_db_connection(schema='') as con2:
+        with get_db_connection(schema='', local=local) as con2:
```

Comment:
@hugwuoke I had another bug here which may have caused your issue. It was reading and updating  certain variables from the production database rather than local. Bit mistake on my part, but at least it wasn't affecting the production database in any other way.